### ISSUE TITLE
Technical Item: Create the Common Image reference fields

### ISSUE THREAD

**Issue by bryanpizzillo** [NCIOCPL/cgov-digital-platform#421]
## Description 

Create the common image reference fields:
* [Article Image](https://github.com/NCIOCPL/cgov-digital-platform/wiki/Common-Fields#article-image)
* [Promotional Image](https://github.com/NCIOCPL/cgov-digital-platform/wiki/Common-Fields#promotional-image)

## Tasks
*  Create the following: 
    * A view for the Image browser
    * The Image Browser configuration
    * The field storage in cgov_image module 
    * Tests for:
       * The field storage
       * Tests for creating the a cgov_image top reference 
*  Enable the following modules:
    * entity_browser
    * ctools (Technically only needed for local dev to create entity browsers within the Admin UI)
*  Configure the:
    * ??
* Documentation on how to setup the field on a content type

**Comment by bryanpizzillo** [comment]
This is going to be a MVP for the image browser. I am keeping this stock Entity Browser because while there is a Media Entity Browser module that we used in the PoC, that is more of a pre-configured browser, which we hacked up the configuration for.  I don't want to figure out how we can manage their version and our overrides, and where our override configs should go. 

I propose we just use it OOB for now, and apply similar CSS & JS post launch to make it prettier.
